ctrd/client.go

```diff
@@ -384,15 +384,15 @@ func (c *Client) collectContainerdEvents() {
 		}
 
 		switch e.Topic {
-		case ContainersDeleteEventTopic:
-			cDelEvent, ok := out.(*eventstypes.ContainerDelete)
+		case TaskExitEventTopic:
```

Comment:
Need know the exit code when a container exited, so we can get the task exit code from containerd `task/exit` event